python3-modules.json

```diff
@@ -0,0 +1,221 @@
+{
+    "name": "python3-modules",
```

Comment:
there are many packages listed more than once in that manifest.

And I'm pretty sure pyqt5 will not build. Why did you change the way it was before?

Reply:
pyqt5 is anymore available on riverbankcomputing